src/helpers/graphHelper.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+ * See LICENSE in the project root for license information.
+ */
+
+/* global OfficeRuntime */
+import { dialogFallback } from "./fallbackAuthHelper";
+import { handleClientSideErrors } from "./../../node_modules/office-addin-sso/lib/error-handler";
+import { MSGraphHelper } from "./../../node_modules/office-addin-sso/lib/msgraph-helper";
```

Comment:
So this file is called graphHelper.ts, but it calls a different file called msgraph-helper. This is confusing. This file isn't really about helping with graph calls. It's about using the getAccessToken method and handling it's errors. I think it should be renamed something like ssoAuth.ts.

Reply:
sure we can call it that

Reply:
fixed